.circleci/config.yml

```diff
@@ -2,6 +2,29 @@ version: 2
 orbs:
   coveralls: coveralls/coveralls@1.0.4
 jobs:
+  lint:
+    docker:
+      # specify the version you desire here
+      - image: circleci/node:10.15.3-browsers
+
+      # Specify service dependencies here if necessary
+      # CircleCI maintains a library of pre-built images
+      # documented at https://circleci.com/docs/2.0/circleci-images/
+      # - image: circleci/mongo:3.4.4
+
+    working_directory: ~/user-data-for-fraud-prevention
+
+    steps:
+      - checkout
+
+      # install eslint
+      - run: npm install eslint
```

Comment:
I don't think you need to install eslint packages. They are already being installed as part of npm install step in the build job. You just need to run lint task.

Reply:
Hi, Thanks for the input, I've made the changes...

:)